doc/psc-ide-vim.txt

```diff
@@ -138,3 +138,68 @@ taken.
  :Psearch ident
 <Search for identifier.  This works like the provided 'omnifunc', but with
 less filtering.
+<
+
+SETTINGS                                *purs-settings*
+
+>
+ g:psc_ide_log_level [ident]
+<Controls the verbosity of debug output.
+  0 =       no output
+  1 =       less output
+  2 =       more output
+  3 =       debug outupt
+
+  default is 0
+>
+ g:psc_ide_server_port [ident]
```

Comment:
I think it s
`ident` shouldn't be in brackets here, it's not an optional value as it comes with a default value.

Reply:
Neither it's a list...